stripper/ze_l0v0l_v1_4_csgo2.cfg

```diff
@@ -141,3 +141,17 @@ modify:
 		"OnStartTouch" "lvl4winfixEndRound_CounterTerroristsWin56-1"
 	}
 }
+
+;Fix particles not appearing
```

Comment:
Can you add a loose vscript notice, similar to [this](https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/master/stripper/ze_tyranny2_v1_csgo2.cfg#L68-L71).